at_client/lib/src/util/at_client_validation.dart

```diff
@@ -28,10 +44,10 @@ class AtClientValidation {
           'Invalid TTL value: ${metadata.ttl}. TTL value cannot be less than 0');
     }
     // validate TTB
-    if (metadata.ttb != null && metadata.ttb! < 0) {
-      throw AtKeyException(
-          'Invalid TTB value: ${metadata.ttb}. TTB value cannot be less than 0');
-    }
+    // if (metadata.ttb != null && metadata.ttb! < 0) {
```

Comment:
Why is this code dead, and what's the purpose of leaving it here commented out?

Reply:
Uncommented the code and pushed the changes